docs/commands/New-PesterConfiguration.mdx

```diff
@@ -0,0 +1,240 @@
+
+  Line: Filter by file and scriptblock start line, useful to run parsed tests programatically to avoid problems with expanded names.
+Example: 'C:\tests\file1.Tests.ps1:37'
+  Default value: @()
+
+  FullName: Full name of test with -like wildcards, joined by dot.
+Example: '*.describe Get-Item.test1'
+  Default value: @()
+
+CodeCoverage:
+  Enabled: Enable CodeCoverage.
+  Default value: $false
+
+  OutputFormat: Format to use for code coverage report.
+Possible values: JaCoCo, CoverageGutters
+  Default value: 'JaCoCo'
+
+  OutputPath: Path relative to the current directory where code coverage report is saved.
+  Default value: 'coverage.xml'
+
+  OutputEncoding: Encoding of the output file.
+  Default value: 'UTF8'
+
+  Path: Directories or files to be used for codecoverage, by default the Path(s) from general settings are used, unless overridden here.
+  Default value: @()
+
+  ExcludeTests: Exclude tests from code coverage.
+This uses the TestFilter from general configuration.
+  Default value: $true
+
+  RecursePaths: Will recurse through directories in the Path option.
+  Default value: $true
+
+  CoveragePercentTarget: Target percent of code coverage that you want to achieve, default 75%.
+  Default value: 75
+
+  SingleHitBreakpoints: Remove breakpoint when it is hit.
+  Default value: $true
+
+TestResult:
+  Enabled: Enable TestResult.
+  Default value: $false
+
+  OutputFormat: Format to use for test result report.
+Possible values: NUnitXml, NUnit2.5 or JUnitXml
+  Default value: 'NUnitXml'
+
+  OutputPath: Path relative to the current directory where test result report is saved.
+  Default value: 'testResults.xml'
+
+  OutputEncoding: Encoding of the output file.
+  Default value: 'UTF8'
+
+  TestSuiteName: Set the name assigned to the root 'test-suite' element.
+  Default value: 'Pester'
+
+Should:
+  ErrorAction: Controls if Should throws on error.
+Use 'Stop' to throw on error, or 'Continue' to fail at the end of the test.
+  Default value: 'Stop'
+
+Debug:
+  ShowFullErrors: Show full errors including Pester internal stack.
+  Default value: $false
+
+  WriteDebugMessages: Write Debug messages to screen.
+  Default value: $false
+
+  WriteDebugMessagesFrom: Write Debug messages from a given source, WriteDebugMessages must be set to true for this to work.
+You can use like wildcards to get messages from multiple sources, as well as * to get everything.
+  Default value: @('Discovery', 'Skip', 'Filter', 'Mock', 'CodeCoverage')
+
+  ShowNavigationMarkers: Write paths after every block and test, for easy navigation in VSCode.
+  Default value: $false
+
+  ReturnRawResultObject: Returns unfiltered result object, this is for development only.
+Do not rely on this object for additional properties, non-public properties will be renamed without previous notice.
+  Default value: $false
+
+Output:
+  Verbosity: The verbosity of output, options are None, Normal, Detailed and Diagnostic.
+  Default value: 'Normal'
+
+## EXAMPLES
```

Comment:
````suggestion
  Default value: 'Normal'
```

Adding codeblock
## EXAMPLES
````

Reply:
Looks like I snuck in a easter egg here ("Adding codeblock"), well well  it'll disappear on the next update 😄

Reply:
Good egg and good to sea that you folks were able to update the help. It's a relief to know that more people can do it.

Much appreciate it.